docs/source/release_notes/release-notes.rst

```diff
@@ -32,6 +32,24 @@ Changelog
 .. <<<<------------------------------------------------------------------------------------------------------------->>>>
 
 =======
+develop
```

Comment:
I wasn't quite sure what you usually do here, but I did notice two 0.27.0 entries below.